libs/filabridge/src/SibGenerator.cpp

```diff
@@ -16,19 +16,22 @@
 
 #include "private/filament/SibGenerator.h"
 
+#include "private/filament/Variant.h"
+
 #include <backend/DriverEnums.h>
 
 #include <private/filament/SamplerInterfaceBlock.h>
 
 namespace filament {
 
-SamplerInterfaceBlock const& SibGenerator::getPerViewSib() noexcept {
+SamplerInterfaceBlock const& SibGenerator::getPerViewSib(uint8_t variantKey) noexcept {
     using Type = SamplerInterfaceBlock::Type;
     using Format = SamplerInterfaceBlock::Format;
     using Precision = SamplerInterfaceBlock::Precision;
 
-    // TODO: ideally we'd want this to be constexpr, this is a compile time structure
-    static SamplerInterfaceBlock sib = SamplerInterfaceBlock::Builder()
+    // TODO: ideally we'd want these to be constexpr, these are compile time structures.
+
+    static SamplerInterfaceBlock sibPcf = SamplerInterfaceBlock::Builder()
```

Comment:
It might be worth having a shared function that fills in the non-specific parameters so we don't have to maintain two lists. Seems like an easy thing to mess up at some point :)

Reply:
Yeah good point